docs/synapse/userguides/ug014_storm_ref_filter.rst

```diff
@@ -0,0 +1,20 @@
+Storm Reference - Filter Operators and Comparators
+==================================================
+
+Todo
+
+See the `Storm Syntax Conventions`__ for an explanation of the usage format used below.
+
+Where specific query examples are given, they are commonly provided in pairs using operator syntax followed by the equivalent macro syntax (if available).
+
+filter()
```

Comment:
not-usable afaik

Reply:
I would note that `filter` is not actually an operator and that users need to use `<plus or minus><prop><optional comparator and valu>` or `<plus or minus><#tagname>`.